src/pow.cpp

```diff
@@ -29,12 +29,15 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
         if (pblock->GetBlockTime() > pindexLast->GetBlockTime() + params.nPowTargetSpacing*2){
             return nProofOfWorkLimit;
         }
+        // Well, actually, it wasn't ever implemented properly in LBRYcrd.
+        // It doesn't work without the missing "else" statement found in upstream bitcoin.
+        // And that statement doesn't work correctly with our DifficultyAdjustmentInterval == 1.
+        // Hence, we are killing it at block 1100000.
     }
```

Comment:
I agree it hasn't worked properly, but I think returning the difficulty back to something sane is better than letting testnet potentially get stuck.  The entire point of the testnet is to make testing easier/possible, so the extra headache if ever stuck is just an added barrier.  Doing nothing arguably allows easy testing (and mining) to continue as well.

Reply:
It's allowed to move by an eighth (12.5%) every block. You can travel a long way in half an hour -- something most people could tolerate. Something sane might be the median of the past 50 blocks; it could get complicated. 

Judging form the numbers I was seeing, though, there are miners going in and out of this often. I don't know why that is. I know that our testnet servers are on shared machines; maybe their resources fluctuate significantly. Or maybe the addition of a third person is sporadic but not uncommon.